### PR TITLE
New worker job: Find Largest Segment ID

### PR DESCRIPTION
**Pull request by fm3**
wk side for https://github.com/scalableminds/webknossos-worker/pull/114

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Edit a dataset. Should look like this (if the largest segment id is empty):
image
- Click the Detect button
image
image

- Should show a toast:
image
- After reloading:
image
(the message would typically be something like `Updated largest segment id from 1 to 1470623.`)

It would be even better if the largest segment id was automatically updated in the form, but for long running jobs this wouldn't really matter (as one can close the tab) and the interface of the API doesn't really lend itself to that currently.

### Issues:
- follow-up for #6039 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

**Comment by fm3** [comment]
@philippotto I adapted the job list view so that it displays the returned result message in the actions column. I’m aware that it does not really fit the column title but I don’t think creating a new column is worth it here. What do you think? Also, where do you think would be a good place for a button to trigger this job? Maybe you could have a look at the front-end part here?

**Comment by philippotto** [comment]
I added some prototypical code to trigger the job (mainly so that I could test the worker). however, for a proper integration, I'd like to wait until #6485 is merged.

**Comment by philippotto** [comment]
cc @fm3 the frontend is ready for review. you might want to request a review for the backend, too :)

File: frontend/javascripts/dashboard/dataset/dataset_settings_data_tab.tsx
Comment (daniel-wer) [inline frontend/javascripts/dashboard/dataset/dataset_settings_data_tab.tsx:338]:
```suggestion
    failureMessage: "The computation of the largest segment id for this dataset didn't finish properly.",
```

File: frontend/javascripts/admin/job/job_hooks.ts
Comment (daniel-wer) [inline frontend/javascripts/admin/job/job_hooks.ts:35]: Shouldn't `(job) => findJobPred(job)` be true for all jobs in `potentialJobs`?

**Review by daniel-wer — COMMENTED** [review]
Great abstraction and UX as always! :)

I did not test this PR, mainly because I don't have a running worker setup locally, atm.

File: frontend/javascripts/admin/job/job_hooks.ts
Comment (philippotto) [reply frontend/javascripts/admin/job/job_hooks.ts:10]: Good point! I created a new issue: https://github.com/scalableminds/webknossos/issues/6599
I think, refactoring existing code might not be worth it (especially, since the hook cannot be used in class-components), but especially for new job-integrations, we should keep this in mind :)

**Comment by fm3** [comment]
I can’t approve my own PR but I’d say if philipp tested that the backend passes the parameters through so that the job runs as expected, the backend does not need further review :) :shipit:

**Review by philippotto — APPROVED** [review]

**Comment by fm3** [comment]
I did not look at the front-end code in deatil, but one question just to make sure: is this disabled if jobs are not enabled for the instance and the dataset? (I’m asking with https://github.com/scalableminds/webknossos/issues/6582 in mind)

**Pull request merged**
